tests/acceptance/features/webUIFiles/breadcrumb.feature

```diff
@@ -0,0 +1,26 @@
+Feature: access breadcrumb
+  As a user
+  I want to browse to parent folders using breadcrumb
+  So that I can access resources with ease
+
+  Background:
+    Given user "user1" has been created with default attributes
+
+  @issue-2538
+  Scenario: Check breadCrumb for folder one level below the root folder when rootFolder is set
+    Given rootFolder has been set to "" in config.json
```

Comment:
```suggestion
    Given the rootFolder has been set to ""
```
I dont think it is a better idea to give file name in the feature file what do you think?

Reply:
what about calling it "phoenix config file"?